stream-chat-android-compose-sample/src/main/java/io/getstream/chat/android/compose/sample/MessagesActivity.kt

```diff
@@ -164,15 +164,15 @@ class MessagesActivity : AppCompatActivity() {
                 .wrapContentHeight(),
             viewModel = composerViewModel,
             integrations = {},
-            input = {
+            input = { value, attachments, action ->
```

Comment:
Instead of directly observing state within the component, we can propagate it down to make it fully stateless.